sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S4055.json

```diff
@@ -0,0 +1,108 @@
+{
+"issues":  [
```

Comment:
Have you seen cases which we could potentially not report on? For example, formatting strings, or only special characters? Would it be useful to revise this rule in the future, or are most cases right?

(I've checked few at random and they were all real issues.)